### DISCUSSION TITLE
Compatible with OS 4.x on Cloud Gateway Ultra/Max (2024) ?

### DISCUSSION THREAD

**Discussion by r3n3x** [SierraSoftworks/tailscale-udm#94] [Q&A] [answered]
Great project!
Before I try it out, can I ask whether you already know...

-> Would this work on the UCG Ultra running OS 4.x? (4.0.6 for me)

Thanks

**Comment by notheotherben** [comment, marked as answer]
It should work on OS 4.x without any issues, but do shout if you find an issue.
